exasol_integration_test_docker_environment/lib/test_environment/parameter/docker_db_test_environment_parameter.py

```diff
@@ -1,11 +1,14 @@
 import luigi
+from enum import Enum
 from luigi import Config
 
+DockerAccessMethod = Enum("DockerAccessMethod", ["DOCKER_EXEC", "SSH"])
```

Comment:
If it is not dynamically generated I recommend you stick with the class based Syntax.

```
class DockerAccessMethod(Enum):
     DOCKER_EXEC = auto()
     SSH = auto()
```

Reply:
I will do so.